install.sh

```diff
@@ -55,7 +55,6 @@ export MY_CONDA_DIR=$conda_prefix
 export PATH=$MY_CONDA_DIR/bin:$PATH
 
 echo "==== Create 'dgfem' conda environment"
-conda init
 
```

Comment:
The info at the bottom of this file probably needs to be adjusted as well.

Reply:
Good catch, thanks!